dbms/src/Storages/Transaction/PartitionStreams.h

```diff
@@ -17,6 +17,7 @@
 #include <Storages/ColumnsDescription.h>
 #include <Storages/DeltaMerge/DeltaMergeDefines.h>
 #include <Storages/Transaction/DecodingStorageSchemaSnapshot.h>
+#include <Storages/Transaction/TMTContext.h>
```

Comment:
Using forward declaration `class TMTContext;` instead of including this header file is better here